src/GraphQL.Serialization.Newtonsoft/GraphQL.Serialization.Newtonsoft.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>GraphQL</RootNamespace>
```

Comment:
I think that it will be better to use different namespace - GraphQL.Serialization.Newtonsoft.

Reply:
Done.